source/people/main.rst

```diff
@@ -106,7 +97,9 @@ None at this time.
 Alumni
 -------------
 * `Tae Wook Ahn <http://www.oaep.go.th/index_en.php>`_, MS(2010), Exelon Nuclear
+* Elliott Biondo, PhD(2016), Idaho National Laboratory
```

Comment:
Are these switched? I thought Elliott was going to ORNL and Robert was going to INL